apps/meteor/server/models/raw/AppLogsModel.ts

```diff
@@ -13,7 +13,9 @@ export class AppsLogsModel extends BaseRaw<any> implements IAppLogsModel {
 	}
 
 	async resetTTLIndex(expireAfterSeconds: number): Promise<void> {
-		await this.col.dropIndex('_updatedAt_1');
+		if (await this.col.indexExists('_updatedAt_1')) {
```

Comment:
Running the new migration would trigger an error here due to the attempt of removing an unexistent `_updatedAt_1` index in AppsLogs, so I also needed to check whether this index exists to avoid throwing this error :man_shrugging: